doc/util/versionButton.php

```diff
@@ -0,0 +1,142 @@
+
+<style>
+#versionButton {
+  padding: 4px;
+  border: none;
+  cursor: pointer;
+}
+
+/* Blue */
+.currentVersion {
+  background-color: #36acf9;
+  color: white;
+}
+.currentVersion:hover, .currentVersion:focus {
+  background-color: #2f95d8;
+}
+
+/* Orange */
+.oldVersion {
+  background-color: #f49542;
+  color: white;
+}
+.oldVersion:hover, .oldVersion:focus {
+  background-color: #d68137;
+}
+
+/* Yellow */
+.preRelease {
+  background-color: #f4df42;
+  color: black;
+}
+.preRelease:hover, .preRelease:focus {
+  background-color: #d3c139;
+  color: black;
+}
+
+#dropContainer {
+  position: relative;
+  display: inline-block;
+  margin-bottom:10px;
+}
+
+#dropDiv {
+  display: none;
+  position: absolute;
+  background-color: #f9f9f9;
+  min-width: 160px;
+  overflow: auto;
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+  z-index: 1;
+  height: 150px;
+  right: 0;
+}
+
+#dropDiv a {
+  color: black;
+  padding: 12px 16px;
+  text-decoration: none;
+  display: block;
+}
+
+.dropdown a:hover {background-color: #f1f1f1}
+
+</style>
+<div id="dropContainer">
+  <button id="versionButton" onclick="toggleDropDown()"></button>
+  <div id="dropDiv">
+  </div>
+</div>
+<script type="text/javascript">
+
+var dropDiv = document.getElementById("dropDiv");
+
+function toggleDropDown() {
+  var dsp = dropDiv.style.display;
+  if (dsp != "block") {
+    dropDiv.style.display = "block";
+  } else {
+    dropDiv.style.display = "none";
+  }
+}
+
+var chplTitle = "<?php echo "$chplTitle";?>";
+
+// Note: assumes second element is most-recent release
+var chplVersions = [
```

Comment:
This is something we'll need to update every release. Do you think it's possible to generate it automatically instead?

Reply:
I think point releases make this difficult to automate, but I'm open to hearing ideas. I just did this manually so that we can make progress.

Reply:
I agree that desire for auto-updating shouldn't hold up getting a first draft in.  I also don't think doing this update is that much worse than other documentation and version number-related updates that we already require and could be added to the list.